actions/copy/copy.js

```diff
@@ -0,0 +1,50 @@
+/* ************************************************************************
+* ADOBE CONFIDENTIAL
+* ___________________
+*
+* Copyright 2023 Adobe
+* All Rights Reserved.
+*
+* NOTICE: All information contained herein is, and remains
+* the property of Adobe and its suppliers, if any. The intellectual
+* and technical concepts contained herein are proprietary to Adobe
+* and its suppliers and are protected by all applicable intellectual
+* property laws, including trade secret and copyright laws.
+* Dissemination of this information or reproduction of this material
+* is strictly forbidden unless prior written permission is obtained
+* from Adobe.
+************************************************************************* */
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+const openwhisk = require('openwhisk');
+const { getAioLogger } = require('../utils');
+
+// This returns the activation ID of the action that it called
+function main(args) {
+    const logger = getAioLogger();
+    let payload;
+    try {
+        const { spToken, adminPageUri, projectExcelPath } = args;
```

Comment:
As discussed, this should be revisited once `loc` has established an auth flow with sharepoint. Ideally, a user's SP token does not need to be passed in cleartext to the API call.